apps/balancer-tools/pre-build.js

```diff
@@ -70,7 +70,7 @@ const EXISTING_NETWORKS = {
 const POOLS_WITHOUT_GAUGE_QUERY = `
 query PoolsWherePoolType($skip: Int!) {
   pools(
-    where: { totalLiquidity_gt: 5000 }
+    where: { totalLiquidity_gt: 10000 }
```

Comment:
this should probably be removed altogether

Reply:
We have some validations on `gauge.ts` and `validate.ts` that use `POOLS_WITHOUT_GAUGES`. I will think about how we can remove this. My only fear is that it will try to perform the validations before `APRPools` query is done

Reply:
gotcha. fwiw this could have virtually all pools (so no totalLiquidity filter) if it's only being used for getting the pool created date